dbos/templates/hello/__package/main.py

```diff
@@ -28,7 +77,7 @@ def example_transaction(name: str) -> str:
         )
         .returning(dbos_hello.c.greet_count)
     )
-
     greet_count = DBOS.sql_session.execute(query).scalar_one()
-    DBOS.logger.info(f"{name} greet_count: {greet_count}")
-    return name + str(greet_count)
+    greeting = f"Greetings, {name}! You have been greeted {greet_count} times."
+    DBOS.logger.info(greeting)
+    return greeting
```

Comment:
On the TS side, I also return the HTML with links back to the doc here (after the greeting counter).

For a user who just deployed their first app and just got to this page - consider that there's nothing else on the screen. A link that says "go back to the doc.. " may be a good guide for what to do next

Reply:
I'll try it and see how it looks.

I like the idea of the templates serving HTML, but it's really important that they look good because they'll be people's first introduction to DBOS.  That means both the served HTML and the code should look good (and unfortunately those goals are at odd, because templated HTML in the code does not look good). I think the new CSS styling is a big improvement for the readme, though we can iterate on and improve it. I'll see what can be done here while keeping the code clear.

Reply:
Indeed... I fear some ugliness cannot be created or destroyed but maybe it can be moved to a more suitable place :)